src/validateEntityWithSchema.ts

```diff
@@ -0,0 +1,25 @@
+import { IntegrationSchema } from '.';
+
+/**
+ * Validates an entity using the data model schemas, throwing an error when
+ * validation fails. It is expected that this will only be used during
+ * development, not in production!
```

Comment:
```suggestion
 * validation fails.
```

I think when to validate (call this function) is not a concern of this library.

Reply:
Oops, didn't read those docs, mb. :grimacing: willfix